wiki/macbookPro/OpenMP.md

```diff
@@ -0,0 +1,7 @@
+Use Homebrew to get g++ with OpenMP:
```

Comment:
something @harterj and I have been discussing are these mac package installer deals. There are a number of them. ID'ing which ones are better or worse than others at things would be useful to know. Personally, I don't have any of them. @harterj has a few installed on his machine.